moveit_ros/moveit_servo/test/unit_test_servo_calcs.cpp

```diff
@@ -370,7 +370,7 @@ TEST_F(ServoCalcsTestFixture, TestEnforceVelLimits)
   for (size_t i = 0; i < 7; ++i)
   {
     // We need to check vs radians-per-loop allowable rate (not rad/s)
-    EXPECT_GE(desired_velocity[i], -1 * vel_limits[i] * servo_calcs_->parameters_->publish_period);
+    EXPECT_LE(desired_velocity[i], -1 * vel_limits[i] * servo_calcs_->parameters_->publish_period);
```

Comment:
This should be "greater than" because we're dealing with negative numbers

Reply:
Yeah, you are right I thought maybe the tests were wrong since the outcome was vice versa but Henning helped me figure out. I'll change it back

Reply:
I went ahead and reverted the changes to this file so we can see if CI passes reliably with moveit_resources 63

Reply:
Sure, thanks a lot